.gitignore

```diff
@@ -9,14 +9,7 @@ dist-qt/
 enc_temp_folder/
 .cache/
 */out/
-
-# Exclude original game assets
-attic/
```

Comment:
These should definitely stay in. The leaked source code of the game was a full project dump, including a ton of old and unused files under "attic", as well as all of the games assets under "assets". Don't even know what the "scripts" are for, but certainly not needed to modify just the game. The safest way of distributing something like that is to make 100% sure no game assets are included anywhere, so that the .exe alone doesn't do enough to count as "working" and hence not allowing to bypass "buying the game".

As for potential additions to the assets, as with your included "StarGlue" test mod to make some asset changes, these should better be distributed and managed seperately - be it on another branch or repo alltogether.